reference/ps-modules/PSScriptAnalyzer/PSScriptAnalyzer.md

```diff
@@ -1,10 +1,10 @@
 ---
 Download Help Link: https://aka.ms/ps-modules-help
-Help Version: 1.21.0
+Help Version: 1.22.0
 Locale: en-US
 Module Guid: d6245802-193d-4068-a631-8863a4342a18
 Module Name: PSScriptAnalyzer
-ms.custom: PSSA v1.21.0
+ms.custom: PSSA v1.22.0
 ms.date: 10/07/2021
```

Comment:
what is the point of those `ms.date` fields if they never get updated? Can we not just remove them?

Reply:
They are used in our publishing system to measure "freshness". The date should get updated if there are significant changes to an article.